jena-fuseki2/jena-fuseki-webapp/src/main/java/org/apache/jena/fuseki/cmd/DSGSetup.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.jena.fuseki.cmd;
+
+import jena.cmd.CmdException;
+import org.apache.jena.atlas.io.IO;
+import org.apache.jena.fuseki.mgt.Template;
+import org.apache.jena.fuseki.system.spot.TDBOps;
+
+public class DSGSetup {
```

Comment:
Would it make sense to combine `cmds/DSGSetup` and this `DSGSetup`? Or maybe add a comment saying that that one is for `ServerConfig`, and this one for `FusekiArgs`?

Reply:
Good point. renamed one and also made them package scoped.